src/main/resources/com/google/api/codegen/go/discovery_fragment.snip

```diff
@@ -113,35 +148,76 @@ func main() {
 @end
 
 @private initialize(method)
-  @join param : context.getFlatMethodParams(method) on BREAK.add(BREAK)
-    @let signatureType = context.getApiaryConfig.getType(method.getRequestTypeUrl), \
-        paramField = context.getField(signatureType, param), \
-        paramValue = context.typeDefaultValue(signatureType, paramField), \
-        paramDescription = context.getApiaryConfig.getDescription(signatureType.getName, param)
-      {@description(paramDescription)}
-      {@unclashVar(param)} := {@paramValue} // {@TODO()} Update placeholder value.
+  @let methodName = context.lowerCamelToUpperCamel(context.getMethodName(method)), \
+       signatureType = context.getApiaryConfig.getType(method.getRequestTypeUrl), \
+       params = context.getFlatMethodParams(method)
+    @if params
+      // {@TODO()} Change placeholders below to desired parameter values for the `{@methodName}` method:
+
+      @join param : params
+        @let paramField = context.getField(signatureType, param), \
+             paramValue = context.typeDefaultValue(signatureType, paramField), \
+             paramDescription = context.getDescription(signatureType.getName, param)
+          {@description(paramDescription)}
+          {@unclashVar(param)} := {@paramValue}
+
+        @end
+      @end
     @end
   @end
+  @if context.hasRequestField(method)
+
+    {@requestBodyName(method)} := {@context.getApi.getName}.{@context.getRequestField(method).getTypeUrl}{
+      @if context.isPatch(method)
+        // {@TODO()} Add desired fields of the `{@requestBodyName(method)}` variable to be changed.
+      @else
+        // {@TODO()} Add desired fields of the `{@requestBodyName(method)}` variable.
+      @end
+    }
+
+  @end
 @end
 
-@private callArg(method)
-  @if context.hasRequestField(method)
-    @if context.getFlatMethodParams(method)
-      {@callArgNoBody(method)}, &{@context.getApi.getName}.{@context.getRequestField(method).getTypeUrl}{
-        // {@TODO()} Fill required fields.
-      }
+@private arguments(method)
+  @let params = context.getFlatMethodParams(method)
+    @if context.hasRequestField(method)
+      @if params
+        {@argumentList(params)}, &{@requestBodyName(method)}
+      @else
+        {@requestBodyName(method)}
+      @end
     @else
-      &{@context.getApi.getName}.{@context.getRequestField(method).getTypeUrl}{
-        // {@TODO()} Fill required fields.
-      }
+      {@argumentList(params)}
     @end
-  @else
-    {@callArgNoBody(method)}
   @end
 @end
 
-@private callArgNoBody(method)
-  @join param : context.getFlatMethodParams(method) on ", "
+@private clientName()
```

Comment:
I see this reverses part of https://github.com/googleapis/toolkit/pull/117, but I don't understand why that change was made. If there's a good reason, we should consider a change across all languages.